spec/factories/tasks.rb

```diff
@@ -5,5 +5,14 @@
     from = Date.parse("2019/08/01")
     to   = Date.parse("2019/12/31")
     deadline { Random.rand(from..to) }
+    association :project
+
+    trait :status_done do
+      status { :done }
+    end
+
+    trait :completion_yesterday do
+      completion_date { Time.current.yesterday }
+    end
```

Comment:
タスクのステータスが完了であれば、完了日が登録されている仕様なので、テストデータは「ステータスが完了かつ完了日が登録済」の状態で作りたいですね。
```suggestion
    trait :done do
      status { :done }
      completion_date { Time.current.yesterday }
    end
```

Reply:
c1ae7b2 でtraitを一つに統合し、「ステータスが完了かつ完了日が登録済」という状態へ変更しました。